entrypoint.sh

```diff
@@ -50,5 +51,20 @@ fi
 
 export OKTETO_DISABLE_SPINNER=1
 
-echo running: okteto pipeline deploy --name "${name}" --branch="${branch}" --repository="${GITHUB_SERVER_URL}/${repository}" ${params} --wait
-okteto pipeline deploy --name "${name}" --branch="${branch}" --repository="${GITHUB_SERVER_URL}/${repository}" ${params} --wait
+output=$(cat <<-END
+    okteto pipeline deploy
+    --name "${name}"
+    --branch="${branch}"
+    --repository="${GITHUB_SERVER_URL}/${repository}"
+    ${params}
+END
+)
+
+if [ -n "$filePath" ]; then
```

Comment:
Could you follow the same approach than other parámeters? that means doing something like:
```
if [ ! -z "$filePath" ]; then
params="${params} -f "${filePath}""
fi
```
also move it before the line `output=...`